### PR TITLE
[1.x] Add vertical scroll to modal for blade and inertia

### PR DESCRIPTION
**Pull request by marcoT89**
This enables the vertical scroll for long modals, still working for small ones and still centralized.
Also reduces a little bit the way of setting the `$maxWidth` variable. It's not a php8 pattern matching
but it's very similar.

Before:
Screen Recording 2020-12-04 at 10 09 28 PM

After:
Screen Recording 2020-12-04 at 10 12 28 PM

### CODE REVIEW

File: resources/views/components/modal.blade.php [hunk 1]
```diff
@@ -3,24 +3,13 @@
...
-        $maxWidth = 'sm:max-w-2xl';
-        break;
-}
+$maxWidth = [
+    'sm' => 'sm:max-w-sm',
+    'md' => 'sm:max-w-md',
+    'lg' => 'sm:max-w-lg',
+    'xl' => 'sm:max-w-xl',
+    '2xl' => 'sm:max-w-2xl',
+][$maxWidth ?? '2xl'];
```
Comment (bonzai) [inline resources/views/components/modal.blade.php:12]: Please revert this change, because it's a breaking change.

If someone uses a name that is not defined, he will get an exception:

```blade
<x-jet-modal :maxWidth="XX" />

{{-- ErrorException --}}
{{-- Undefined array key "XX" --}}
```

**Comment by nicotravassos** [comment]
Thank you @marcoT89

**Comment by taylorotwell** [comment]
I've applied these changes in master branch.
